modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/auto/AutoConverter.kt

```diff
@@ -194,8 +217,14 @@ object AutoConverter {
      * Use the drawable id so Proguard doesn't remove the asset in the production build.
      */
     fun getBitmapUri(@DrawableRes drawable: Int, context: Context): Uri {
-        val drawableName = context.resources.getResourceEntryName(drawable)
-        return Uri.parse("android.resource://" + context.packageName + "/drawable/" + drawableName)
+        val resources = context.resources
+        // This is an example of the URI android.resource://au.com.shiftyjelly.pocketcasts/drawable/auto_folder_01
+        return Uri.Builder()
+            .scheme(ContentResolver.SCHEME_ANDROID_RESOURCE) // android.resource
+            .authority(resources.getResourcePackageName(drawable)) // au.com.shiftyjelly.pocketcasts
+            .appendPath(resources.getResourceTypeName(drawable)) // drawable
+            .appendPath(resources.getResourceEntryName(drawable)) // auto_folder_01
+            .build()
```

Comment:
This isn't really related to the change, but I thought it was a better way to do it, rather than building up a string.

Reply:
Yeah, this is a nice change. I wanted to do it while working on unifying artwork loading but forgot. Thanks!